mongodbatlas/resource_mongodbatlas_custom_db_role_test.go

```diff
@@ -556,8 +731,6 @@ func testAccMongoDBAtlasCustomDBRolesConfigBasic(projectID, roleName, action, da
 }
 
 func testAccMongoDBAtlasCustomDBRolesConfigWithInheritedRoles(projectID string, inheritedRole []matlas.CustomDBRole, testRole *matlas.CustomDBRole) string {
-	log.Printf("LOG___ inheritedRole: %#+v\n", inheritedRole[1])
```

Comment:
Why this change?

Reply:
Test logs shouldn't be present in the code, it's not necessary